examples/pytorch/text-generator/README.md

```diff
@@ -33,11 +30,7 @@ class PythonPredictor:
         return self.tokenizer.decode(prediction[0])
 ```
 
-Here are the complete [Predictor docs](../../../docs/deployments/realtime-api/predictors.md).
-
-<br>
-
-## Specify your Python dependencies
+## Specify Python dependencies
 
 Create a `requirements.txt` file to specify the dependencies needed by `predictor.py`. Cortex will automatically install them into your runtime once you deploy:
 
```

Comment:
Not in the diff, but it's still in the README, what do you think of freezing the `torch` version?